packages/node/src/utils/commonjs.ts

```diff
@@ -1,4 +1,4 @@
 /** Detect CommonJS. */
 export function isCjs(): boolean {
-  return typeof require !== 'undefined';
+  return typeof module !== 'undefined' && typeof module.exports !== 'undefined';
```

Comment:
To be extra safe: let's either try-catch this, or do this:

```suggestion
  return typeof module !== 'undefined' && module && typeof module.exports !== 'undefined';
```

because technically `module` could be null?